pandas/core/generic.py

```diff
@@ -4949,10 +4951,10 @@ def pipe(self, func, *args, **kwargs):
     Returns
     -------
     DataFrame, Series or scalar
-        If DataFrame.agg is called with a single function, returns a Series
-        If DataFrame.agg is called with several functions, returns a DataFrame
-        If Series.agg is called with single function, returns a scalar
-        If Series.agg is called with several functions, returns a Series.
+       - If DataFrame.agg is called with a single function, returns a Series
```

Comment:
Can you use asterisks here instead?

Reply:
Also I think this will fail once #25132 gets merged. Give that a look to see how to handle